entries/p/payeer.com.json

```diff
@@ -6,8 +6,10 @@
       "email",
       "custom-software"
     ],
-    "documentation": "https://payeer.com/en/faq/",
-    "notes": "Receiving codes via SMS cost 0.05$ each message. 2FA is personalizable in account settings.",
+    "custom-software": [
+      "Telegram"
+    ],
+    "notes": "2FA via SMS costs ¢25 USD per message.",
```

Comment:
Created an account and verified this myself.
(I did however forget to take a screenshot before throwing away the account :see_no_evil: )